src/components/PetBreadCrumbs.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {
+  Breadcrumbs,
+  BreadCrumbLinkItem
+} from "@baltimorecounty/dotgov-components";
+
+const PetBreadCrumbs = props => {
+  const { animalName } = props;
+  console.log(props);
+  return (
+    <Breadcrumbs>
+      <BreadCrumbLinkItem
+        text="Departments"
+        link="/departments/animalservices"
+      />
+      <BreadCrumbLinkItem
+        text="Animal Services"
+        link="/departments/animalservices"
+      />
+      <BreadCrumbLinkItem
+        className="breadcrumb-last"
+        text="Adoptable Pets"
+        link="/animalservices/adoption.html"
+      />
+      <BreadCrumbLinkItem
+        className="breadcrumb-last"
+        text={animalName}
+        link="/animalservices/adoption.html"
```

Comment:
The last item in teh breadcrumb is not a link, since you are already on that page. @theilman7 i believe there is a component for the last item?

Reply:
looking through the library it appears you just dont add a link.

Reply:
@martypowell @tmccoy529 There used to be, but since[ pr 207 ](breadcrumb breadcrumb-last) there is not. To maintain consistency with the rest of the site (since the module in SE will _always_ add the last breadcrumb as a link), we could leave the the link to the current page there. On the other hand, if we wanted to do it without the link in this particular instance we could do that as well. Keeping the link would be consistent with the rest of the site, no link would not.

Reply:
> @martypowell @tmccoy529 There used to be, but since [pr 207](https://github.com/baltimorecounty/dotgov-components/pull/207) there is not. To maintain consistency with the rest of the site (since the module in SE will _always_ add the last breadcrumb as a link), we could leave the the link to the current page there. On the other hand, if we wanted to do it without the link in this particular instance we could do that as well. Keeping the link would be consistent with the rest of the site, no link would not.

Okay, confirming with Sam via chat.